client-core/Cargo.toml

```diff
@@ -33,6 +33,8 @@ jsonrpc-core = "13.2"
 log ="0.4.8"
 serde = { version = "1.0", features = ["derive"] }
 tokio="0.1.22"
+tiny-hderive = "0.2.1"
```

Comment:
`tiny-hderive` depends on `libsecp256k1` -- the same cryptography, but a different library from the one used in our codebase... if it is possible to have cryptographic curve-independent HD implementation, it'll be better than to have the same dependency twice